cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,249 @@ Cask Data Application Platform Release Notes
+
+- :cask-issue:`CDAP-6329` - Enable GC logging for cdap services.
+
+- :cask-issue:`CDAP-11448` - Support HDInsight 3.6
+
+- :cask-issue:`CDAP-11839` - Modified the authorization model so that read/write on an entity will not depend on its parent.
+
+- :cask-issue:`CDAP-11899` - Wrangler transform has the capability to track field-level lineage.
+
+- :cask-issue:`CDAP-12068` - New REST API is added to check the status of metrics processor. We can view the topic level processing stats using this endpoint. 
+
+- :cask-issue:`CDAP-12070` - Added option to disable/enable metrics for a program through runtime arguments or preferences. This feature can also be used system wide by enabling/disabling metrics in cdap-site.xml
+
+- :cask-issue:`CDAP-12084` - Allow appending to (or overwriting) a PartitionedFileSet's partitions when using DynamicPartitione
+
+- :cask-issue:`CDAP-12085` - Expose a programmatic API to leverage Hive's functionality to concatenate a partition of a PartitionedFileSet.
+
+- :cask-issue:`CDAP-12094` - Enhanced add field transform plugin to add multiple fields
+
+- :cask-issue:`CDAP-12100` - Removes automatic grant/revoke privileges on CDAP entity creation/deletion.
+
+- :cask-issue:`CDAP-12134` - Applies the new authorization policy for CDAP.
+
+- :cask-issue:`CDAP-12317` - Added a new configuration 'security.authorization.extension.jar.path' in cdap-site.xml which can be used to add extra classpath and is avalible to cdap security extensions
+
+- :cask-issue:`CDAP-12367` - Support authorization on Kerberos principal for impersonation
+
+- :cask-issue:`CDAP-12378` - Workflow now allows adding configurable conditions with the lifecycle methods.
+
+- :cask-issue:`CDAP-4874` - CSD now performs a version compatibility check with the active CDAP Parcel
+
+- :cask-issue:`CDAP-12017` - Added capability to salt the row key of the metrics tables so that writes are evenly distributed and there is no region hot spotting 
+
+
+Bug Fixes
+---------
+
+- :cask-issue:`CDAP-12103` - Removed cdap-etl-realtime artifact
+
+- :cask-issue:`CDAP-12123` - Removed deprecated cdap-etl-batch jar from packaging
+
+- :cask-issue:`CDAP-12290` - Added global "CDAP" config to enable/disable metrics emission from user programs.By default metrics is enabled.
+
+- :cask-issue:`CDAP-9150` - Allow user to override the InputFormat class and OutputFormat class of a FileSet at runtime.
+
+- :cask-issue:`CDAP-12285` - Fixes an issue with the order of HBase compatibility libraries in the class path.
+
+- :cask-issue:`CDAP-9125` - CDAP Sentry Integration does not rely on every user having their own individual group
+
+- :cask-issue:`CDAP-11095` - Added support for a description field in a pipeline config that will be used as the application's description if set.
+
+- :cask-issue:`CDAP-12020` - Reuse network connections for TMS client.
+
+- :cask-issue:`CDAP-12135` - Deprecated createFilter() and added a new isVisible API in AuthorzationEn
```

Comment:
This is incomplete. It should be:

Deprecated createFilter() and added a new isVisible API in AuthorzationEnforcer. Deprecated grant/revoke APIs for EntityId and added new one for Authorizable which support wildcard privileges